docs: Add environments credentials (#1735)
devopness/devopness: docs/environments/credentials/list-credentials.md

---
title: List Credentials
intro: View the list of credentials linked to a project environment.

links:
  overview:
  quickstart:
  previous:
  next: environments/credentials/view-credential
  guides:
  related:
  featured:
---

> 1. Select a `Project` from the Devopness dashboard
> 1. Select an `Environment` linked to the chosen project

Reviewer: Perhaps make it simpler, and follow the patterns already adopted in existing documentation topics, such as [1] and [2]?

[1] https://www.devopness.com/docs/credentials/add-credential
[2] https://www.devopness.com/docs/applications/add-application

```suggestion
1. On Devopness, navigate to a project then select an environment
```

Author: Done :white_check_mark:

1. On the `Environment` page, click the **Credentials** card or click in `VIEW_ALL` button in card.

>   - The list of existing credentials for this environment will be displayed, showing the credential's name, Provider Service, and Provider Type.

Reviewer: @CleytonRR the content of this documentation topic is good. However, it should not be nested under `environments`.

In our docs all resource types (`project`, `credentials`, `applications`, ...) are listed in the top level menu, as you can see in https://www.devopness.com/docs/

<img width="306" alt="image" src="https://github.com/user-attachments/assets/e45886d1-7d32-4976-b02f-15e04e19a07b" />

Nesting `Credentials` under `Environments` tend to make it less intuitive for the readers of our docs when looking for information.

The issue #1209, to be fixed, only requires new `list-credentials` and `view-credentials` to be created under `docs/credentials`, it's that simple :-) 

Could you please update this PR adjusting file paths and `links`?

Author: Alright, I built it this way by our chat in Linkedin and because in pages the require link to documentations is environments/...

list all credentials
![image](https://github.com/user-attachments/assets/635966c2-7d69-4437-ad9d-5911dc8f3984)

credential detail
![image](https://github.com/user-attachments/assets/d6a0818b-9a7f-4d35-9970-cf2ae7a94dc8)

But I full agree with you credentials under Environments is less intuitive.

Author: I fixed it :)
